packages/driver/src/cli.mts

```diff
@@ -92,9 +92,9 @@ async function main(args: string[]) {
   process.exit(0);
 }
 
-async function whichEdgeDbCli() {
+async function whichGelCli() {
   debug("Checking if CLI is in PATH...");
-  const locations = (await which("edgedb", { nothrow: true, all: true })) || [];
+  const locations = (await which("gel", { nothrow: true, all: true })) || [];
```

Comment:
should we check here for both egdedb and cli?

Reply:
Yeah, if you want to update this now, we should fallback like:

```suggestion
  const locations =
    (await which("gel", { nothrow: true, all: true })) ||
    (await which("edgedb", { nothrow: true, all: true })) ||
    [];
```

Reply:
hmm is my reasoning correct: do we have `gel` or `edgedb` depends on what CLI version is installed, or what server version is installed (my CLI usually depends on the local server version I have), so we can use the latest JS driver but still have older (edgedb) CLI installed

Reply:
> do we have gel or edgedb depends on what CLI version is installed, or what server version is installed

It's based on which CLI version you have installed. Newer CLI versions still work with older server versions.

>  we can use the latest JS driver but still have older (edgedb) CLI installed

Yes, that's right! And we should respect that for now: running `npx gel project init` should call `edgedb project init` if you have an older CLI installed.